af_aisf_jobseeker_sync/models/res_partner.py

```diff
@@ -50,6 +50,29 @@ def _aisf_sync_jobseeker(
             res = rask.call(customer_id=int(customer_id))
             end_time = time.time()
             time_for_call_to_rask = end_time - start_time
+        except ValueError as e:
+            error, res = e.args
+            if res.status_code == 404:
+                _logger.info("here")
```

Comment:
Remove this logging or replace it with something relevant (_logger.info("here"))